src/api/spec/features/webui/attributes_spec.rb

```diff
@@ -34,7 +34,7 @@ def add_attribute_with_values(package = nil)
       expect(page).to have_content('Attribute was successfully updated.')
 
       visit index_attribs_path(project: user.home_project_name)
-      expect(page).to have_content("#{attribute_type.namespace}:#{attribute_type.name} test 2, test 1")
+      expect(page).to have_content("#{attribute_type.namespace}:#{attribute_type.name}\t test 2, test 1")
```

Comment:
I would prefer to check here for the columns and not for the whole page. Something like this maybe:

```
within("tr#attribute-values td").first do
  expect(page).to have_content(
end
```

(untested, just as an idea)